site/styles/_meta-sub-item.scss

```diff
@@ -5,10 +5,17 @@
   grid-template-rows: auto minmax(0, 1fr);
   margin-bottom: 15px;
   padding-bottom: 15px;
+  transition: height 0.5s ease;
+  @include media-breakpoint-up(sm) {
+    height: auto !important;
+  }
 
   @include media-breakpoint-up(sm) {
     grid-template-columns: 175px minmax(100px, 1fr);
     grid-template-rows: auto auto minmax(0, 1fr);
+    .s72-toggle {
```

Comment:
This class shouldn't be in this breakpoint block... it should be outside and have it's own breakpoint block!

Reply:
- fixed as part of 'separate div inside `<s72-show-hide-switcher>`' css refactor